pkg/controller/servicebindingrequest/retriever_test.go

```diff
@@ -16,338 +19,52 @@ func TestRetriever(t *testing.T) {
 	ns := "testing"
 	backingServiceNs := "backing-servicec-ns"
 	crName := "db-testing"
-	testEnvVarPrefix := "TEST_PREFIX"
-	emptyEnvVarPrefix := ""
+	// testEnvVarPrefix := "TEST_PREFIX"
```

Comment:
if it is not needed, please remove fully from the source code.

Reply:
Solved in 7b78469dc6672a40d9d898e3fc32bb04d55f5815.